test/e2e/common/node/pod_admission.go

```diff
@@ -67,83 +65,6 @@ var _ = SIGDescribe("PodOSRejection", framework.WithNodeConformance(), func() {
 	})
 })
 
-var _ = SIGDescribe("PodRejectionStatus", func() {
-	f := framework.NewDefaultFramework("pod-rejection-status")
-	f.NamespacePodSecurityLevel = admissionapi.LevelBaseline
-	ginkgo.Context("Kubelet", func() {
-		ginkgo.It("should reject pod when the node didn't have enough resource", func(ctx context.Context) {
-			node, err := e2enode.GetRandomReadySchedulableNode(ctx, f.ClientSet)
-			framework.ExpectNoError(err, "Failed to get a ready schedulable node")
-
-			// Create a pod that requests more CPU than the node has
-			pod := &v1.Pod{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "pod-out-of-cpu",
-					Namespace: f.Namespace.Name,
-				},
-				Spec: v1.PodSpec{
-					Containers: []v1.Container{
-						{
-							Name:  "pod-out-of-cpu",
-							Image: imageutils.GetPauseImageName(),
-							Resources: v1.ResourceRequirements{
-								Requests: v1.ResourceList{
-									v1.ResourceCPU: resource.MustParse("1000000000000"), // requests more CPU than any node has
-								},
-							},
-						},
-					},
-				},
-			}
-
-			pod = e2epod.NewPodClient(f).Create(ctx, pod)
```

Comment:
/cc @SergeyKanzhelev 

`e2epod.NewPodClient(f).Create(ctx, pod)` will mutate the pod spec to set the node name before the pod is created.

I don't know whether it is right to put this test this package. so I move this test into `e2e/node` and change the above code. 
```go
pod, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(ctx, pod, metav1.CreateOptions{})
framework.ExpectNoError(err)
```

Reply:
I didn't know that `e2e/common/node` will be triggered by node e2e tests. Sorry for that.